<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>View</class>
 <widget class="QMainWindow" name="View">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simon Game</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>270</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 50px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #f6f7fa, stop: 1 #dadbde);
    min-width: 80px;
}
</string>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="redButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>113</width>
      <height>111</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(255, 7, 44);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="blueButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>120</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(17, 0, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>120</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 50px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #f6f7fa, stop: 1 #dadbde);
    min-width: 80px;
}
</string>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar{
border: 2x solid grey;
border-radius: 5px;
text-align: center;
}

QProgressBar::chunk{
background-color:#05b8cc;
width: 10px;
margin: 0.0px;
}
</string>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="greenButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0,255,0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="yellowButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>230</y>
      <width>113</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255,255,0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Starting Level</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>430</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>exitButton</sender>
   <signal>clicked()</signal>
   <receiver>View</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>214</x>
     <y>245</y>
    </hint>
    <hint type="destinationlabel">
     <x>198</x>
     <y>413</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
